lib/supavisor/helpers.ex

```diff
@@ -357,4 +357,12 @@ defmodule Supavisor.Helpers do
     Logger.notice("Setting log level to #{inspect(level)}")
     Logger.put_process_level(self(), level)
   end
+
+  @spec peer_ip(:gen_tcp.socket()) :: String.t()
+  def peer_ip(socket) do
+    case :inet.peername(socket) do
+      {:ok, {ip, _port}} -> "#{:inet.ntoa(ip)}"
```

Comment:
```suggestion
      {:ok, {ip, _port}} -> List.to_string(:inet.ntoa(ip))
```

Reply:
No need for dynamic lookup going through `String.Chars`.

Reply:
Yeah, but it is called once before start and doesn't influence performance

Reply:
okay, need to replace 
```
Name                     ips        average  deviation         median         99th %
list to string       12.20 M       81.97 ns ±25614.07%          42 ns          84 ns
# { }                 9.82 M      101.87 ns ±30388.30%          83 ns          84 ns

Comparison:
list to string       12.20 M
# { }                 9.82 M - 1.24x slower +19.89 ns
[os_mon] cpu supervisor port (cpu_sup): Erlang has closed
[os_mon] memory supervisor port (memsup): Erlang has closed
```